src/languages/css.js

```diff
@@ -77,7 +88,12 @@ export default function(hljs) {
       },
       {
         className: 'selector-pseudo',
-        begin: /:(:)?[a-zA-Z0-9\_\-\+\(\)"'.]+/
+        begin: /:(:)?[a-zA-Z0-9\_\-\+"'.]+/
```

Comment:
Is there a reason the CSS grammar isn't using the new shared values?

Reply:
I'm still not 100% convinced it should. Right now because CSS is a very specific grammar (more rules than the other variants) it can live with a ruleset based grammar instead of a list of attributes.  _That means that it gets updates "for free"._ 

**We'll never have an issue filed because CSS is mixing an attribute or missing a pseudo-selector.**  Less maintenance.  This is the same approach we've recommended other places (see the issue regarding __KEYWORDS in C++).  When the lists can get crazy long and be hard to maintain over time (or likely to age poorly) we prefer patterns.

**The BIG difference here though is that we already have to maintain the patterns for the other grammars though.**  So that might be enough to push it over the top and say just use the lists... welcome to thoughts.

Reply:
I'm ok with either choice, I'd just push for consistency. For example, GitHub highlights only known pseudo selectors but it's consistent with highlighting `content` in CSS and Scss

**CSS**

```css
.hello::before {
  border: 1px solid;
  content: '';
  display: block;
}

.hello:toast {
  content: '';
  toast: '';
}
```

**Scss**

```scss
.hello::before {
  border: 1px solid;
  content: '';
  display: block;
}

.hello:toast {
  content: '';
  toast: '';
}
```

Reply:
> GitHub highlights only known pseudo selectors

Perhaps cause that's a much shorter list than attribute names?

>I'm ok with either choice, I'd just push for consistency.

Well the most consistent way would be to use the same lists in ALL 4 of the grammars. At least when update happen they would happen to all 4 at the same time.

Perhaps first someone could see if there is ANYWAY to avoid an attribute list in some of the cleaner grammars, but I think it's going to be difficult.

>  but it's consistent with highlighting content in CSS and Scss

Highlighting content differently in interesting.  We could do that too, but we'd just have to decide which CSS class was appropriate.  I do see how you could argue it's technically different than other attributes.